extensions/blocks/podcast-player/style.scss

```diff
@@ -118,6 +118,18 @@ $player-float-background: $light-gray-200;
 		}
 	}
 
+	// Apply `secondary` color to the podcast title.
+	.jetpack-podcast-player__container:not(.has-secondary) { // no custom -> default color.
+		a.jetpack-podcast-player__podcast-title {
+			margin: 0;
+		}
+	}
```

Comment:
Why does the color being applied make it have a `margin: 0` or not? Shouldn't that `margin: 0` be applied either way?

Reply:
my bad... :-( updating